models/points.go

```diff
@@ -678,6 +678,13 @@ func scanTags(buf []byte, i int, indices []int) (int, int, []int, error) {
 		case tagValueState:
 			state, i, err = scanTagsValue(buf, i)
 		case fieldsState:
+			// Grow our indices slice if needed
```

Comment:
As a newbie to this part of the code, I wasn't sure what `if needed` actually meant. Could we make it more clear? Something like:
```suggestion
			// Grow our indices slice if we had exactly enough tags to fill it
```